src/main/proto/wfa/measurement/internal/kingdom/protocol_data.proto

```diff
@@ -0,0 +1,81 @@
+// Copyright 2021 The Cross-Media Measurement Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+syntax = "proto3";
+
+package wfa.measurement.internal.kingdom;
+
+import "wfa/common/el_gamal_key.proto";
+import "wfa/common/noise_parameters.proto";
+
+option java_package = "org.wfanet.measurement.internal.kingdom";
+option java_multiple_files = true;
+
+// Protocol-specific configuration and parameters for the Liquid Legions V2
+// Protocol.
+message LiquidLegionsV2Data {
+  // Initial configurations
+  LiquidLegionsV2Config config = 1;
+  // ElGamal keys used for a certain report.
+  map<string, wfa.common.ElGamalPublicKey> duchy_keys = 2;
+}
+
+// Protocol-specific configuration for the Liquid Legions V2 Protocol.
+// Used internally in the report details. The fields should be set when a report
+// is created and then immutable.
+message LiquidLegionsV2Config {
+  // The decay rate of the Liquid Legions sketch.
+  double decay_rate = 1;
+
+  // The maximum size of the Liquid Legions sketch.
+  int64 size = 2;
```

Comment:
decay_rate + size = LiquidLegionsSketchConfig?

Reply:
yep. I can create a message to wrap this two fields